src/index.js

```diff
@@ -139,7 +141,7 @@ class Mongo {
     const id = this.options.convertUserIdToMongoObjectId ? toMongoID(userId) : userId;
     const user = await this.findUserById(id);
     if (user) {
-      return user.services.password.bcrypt;
+      return user.services && user.services.password.bcrypt;
```

Comment:
Maybe it's better to use `get(user, 'services.password.bcrypt');` like that event if a user have a social service and no password set it will not throw.